plugins/org.jkiss.dbeaver.ext.tidb/OSGI-INF/l10n/bundle.properties

```diff
@@ -0,0 +1,70 @@
+Bundle-Vendor = JKISS
+Bundle-Name = DBeaver TiDB Support
+
+manager.catalog.name=Catalog manager
+
+tree.databases.node.name=Databases
+tree.databases.node.tip=Server databases
+tree.database.node.name=Database
+tree.tables.node.name=Tables
+tree.tables.node.tip=Tables
+tree.table.node.name=Table
+tree.columns.node.name=Columns
+tree.column.node.name=Column
+tree.constraints.node.name=Constraints
+tree.constraint.node.name=Constraint
+tree.constraint_columns.node.name=Constraint columns
+tree.check.constraints.node.name=Check constraints
+tree.check.constraint.node.name=Check constraint
+tree.foreign_keys.node.name=Foreign Keys
+tree.foreign_key.node.name=Foreign Key
+tree.foreign_key_columns.node.name=Foreign key columns
+tree.references.node.name=References
+tree.reference_key.node.name=Reference Key
+tree.reference_key_columns.node.name=Reference Key columns
+tree.triggers.node.name=Triggers
+tree.triggers.node.tip=Triggers
+tree.trigger.node.name=Trigger
+tree.indexes.node.name=Indexes
+tree.indexes.node.tip=Indexes
+tree.index.node.name=Index
+tree.index_columns.node.name=Index columns
+tree.partitions.node.name=Partitions
+tree.partition.node.name=Partition
+tree.subpartitions.node.name=Subpartitions
+tree.subpartition.node.name=Subpartition
+tree.views.node.name=Views
+tree.views.node.tip=Views
+tree.view.node.name=View
+tree.users.node.name=Users
+tree.users.node.tip=Users
+tree.user.node.name=User
+tree.userGrants.node.name=Grants
+tree.userGrants.node.tip=User permissions grants
+tree.administer.node.name=Administer
+tree.administer.node.tip=Maintenance/Settings
+tree.sessions.node.name=Sessions
+tree.privilege.node.name=Privilege
+tree.user_privileges.node.name=User Privileges
+tree.system_info.node.name=System Info
+tree.system_info.node.tip=Database system information
+tree.session_status.node.name=Session Status
+tree.variable.node.name=Variable
+tree.global_status.node.name=Global Status
+tree.session_variables.node.name=Session Variables
+tree.global_variables.node.name=Global Variables
+tree.engines.node.name=Engines
+tree.engine.node.name=Engine
+tree.charsets.node.name=Charsets
+tree.charset.node.name=Charset
+tree.plugins.node.name=Plugins
+tree.plugin.node.name=Plugin
+tree.collation.node.name=Collation
+tree.events.node.name=Events
+tree.events.node.tip=Events
+tree.event.node.name=Event
+tree.packages.node.name=Packages
```

Comment:
Hm. But you do not read packages in your Catalog class. As procedures.

Reply:
Deleted.